ideas/151-ens-usernames.md

```diff
@@ -0,0 +1,67 @@
+# Username ENS subdomain Registrar
+
+## Preamble
+
+    Idea: 151-ens-usernames
+    Title: Username ENS subdomain Registrar
+    Status: Draft
+    Created: 2017-11-20
+    Replaces: 027-ens-usernames
+
+## Summary
+
+Eliminates the need to copy/scan - and worse, type - long hexadecimal addresses / public keys, by providing an [ENS](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md) subdomain registry and recognition of ENS names in Status for interacting with other people in Status.
+
+## Swarm Participants
+
+- Lead Contributor: Ricardo Guilherme Schmidt
+- Contributor: Richard Ramos
+- UX (Mist): Alexandre Van de Sande 
+- UX (Status): Hester Bruikman
+- PM: Rachel Hamlin
+
+## Product Overview
+
+A bad experience in most cryptosystems, aswell Status, is the requirement of typing or scanning the public keys and addresses to interact with other users in the network.
+ENS disrupt centralized DNS: like DNS resolves into IPs, ENS resolve into ethereum addresses (and more) trustlessly.
+Status would introduce a option for users registering a human readable contact username and its URI would be used in place of QR codes / private keys.
+A small fee is required, so users are buying the subdomain, and therefore could be exchanged with other users, this network effect is ignored by this idea.
+
+### Product Description
+
+Users would be able to:
+- buy a subdomain from one of the ENS domains owned by Status.
```

Comment:
What does having several domains provide us? How will name be displayed then, with full domain?
Are names supposed to provide some uniqueness within status?

Reply:
As it is now we would provide multiple "domain holders", and dont require uniqueness in between them. For finding a frind you need to send the whole domain. We can start the system providing only one domain option and maybe if we find important add later other domains. 
This details would be controlled by an authority, and later by democracy.

Reply:
@3esmit can subdomains be fully customized? Do we pay anything apart from tx costs to register subdomains on domains we own?